### PR TITLE
Navigation adjustment

### PR DESCRIPTION
**Pull request by rubenvolman**
Added status.poap.xyz as Status in top and bottom navigation

fixed some typos.

### CODE REVIEW

**Comment by rubenvolman** [comment]
Status navigation link proposals:
header navigation:
Screenshot 2022-03-16 at 16 23 26

footer navigation:
Screenshot 2022-03-16 at 16 24 08

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 Install dependencies & start:
 
+    nvm use 15
```
Comment (actuallymentor) [inline README.md:7]: Good one. You may also create an `.nvmrc` by running `echo 15 > .nvmrc && git add -f .nvmrc`

**Review by actuallymentor — COMMENTED** [review]
If this PR is complete you may merge. I see some ADA stuff, so feel free to either finish that in a separate PR or add it to this one (if you add it to the title).

File: README.md (hunk 1 above)
Comment (rubenvolman) [reply README.md]: Not ready for this, Netlify running on a different node version then 15.

**Comment by rubenvolman** [comment]
#21 was fixed in this pull request as well.

**Pull request merged**